chevah/compat/nt_filesystem.py

```diff
@@ -772,7 +772,7 @@ def _setOwner(self, path, owner):
                     path,
                     win32security.SE_FILE_OBJECT,
                     win32security.DACL_SECURITY_INFORMATION,
-                    None,
+                    user_sid,
```

Comment:
there might be an error here... but the current failure is in getOwner and not in setOwner

Reply:
True, this is just a leftover from some changes I've tried related to `setOwner`.  I thought it's best to leave this small change in because it clarifies what that field is used for and the needed variable ( `user_id` above) is already set and used. But no strong feelings, I can back this out if you think otherwise.

Reply:
Happy that this was the only thing out-of-order with the above commit, which prevents the failures for `getOwner` in Windows 2019! :-]